src/pages/dashboard/components/Chart.tsx

```diff
@@ -90,6 +90,24 @@ export function Chart(props: Props) {
     return recordIndex;
   };
 
+  const yAxisWidth = useMemo(() => {
+    const properties = ['invoices', 'outstanding', 'payments', 'expenses'];
+
+    const largestTick = chartData.reduce((maxTick, data) => {
+      return properties.reduce((currentMax, property) => {
+        const currentTickLength = formatMoney(
+          Number(data[property as keyof typeof data]) ?? 0,
+          company?.settings.country_id,
+          currency
+        ).toString().length;
+
+        return Math.max(currentMax, currentTickLength);
+      }, maxTick);
+    }, 0);
+
+    return largestTick;
+  }, [chartData]);
+
```

Comment:
@beganovich After conducting research and reviewing the opinions of other individuals on the GitHub issue regarding the same problem, I am quite confident that the package does not provide a solution for this. Therefore, we need to develop a custom solution. Let me know your thoughts.

Reply:
That's fine. @turbo124 can you QA this?